packages/daemon/src/host.js

```diff
@@ -282,23 +313,26 @@ export const makeHostMaker = ({
             return formulaIdentifier;
           }
 
-          const { formulaIdentifier: lookupFormulaIdentifier } =
-            await provideLookupFormula(petNamePath);
-          return lookupFormulaIdentifier;
-        }),
+          // TODO:lookup Check if a formula already exists for the path. May have to be
```

Comment:
Consider an issue link for TODOs.

Reply:
I'm getting rid of the comment, but did create #2098 because we should have an issue anyway.